.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           - 16
         opendds_branch:
           - master
-          - latest-release
+          # - latest-release
```

Comment:
Is this intentional?

Reply:
Yeah, until the v8 changes are in a release, it won't work. So there's a note on the 3.22 release item to come back and uncomment this line.